pybedtools/bedtool.py

```diff
@@ -2445,27 +2479,31 @@ def bedpe_to_bam(self):
 
     @_log_to_history
     @_wraps(prog="clusterBed", implicit="i")
-    def cluster(self):
+    def cluster(self, *args, **kwargs) -> BedTool: # type: ignore
         """
         Wraps  `bedtools cluster`.
         """
 
     @_log_to_history
     @_wraps(prog="unionBedGraphs")
-    def union_bedgraphs(self):
+    def union_bedgraphs(self, *args, **kwargs) -> BedTool: # type: ignore
         """
         Wraps `bedtools unionbedg`.
 
         Warning: using the `header=True` kwarg will result in a file that is
         not in true BED format, which may break downstream analysis.
         """
+        if "header" in kwargs:
```

Comment:
I though it was a good idea to warn the user of the specific case rased where the resulting file is not a true bed file